### PR TITLE
Enable oneSync always

### PR DESCRIPTION
**Pull request by iskakaushik**
This feature has been stabilized and no longer needs a disable flag.

### CODE REVIEW

**Review by serprex — APPROVED** [review]

**Comment by serprex** [comment]
In flowable_core.go you can remove `SessionID == ""` check

File: flow/activities/flowable_core.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgxpool"
-	"github.com/yuin/gopher-lua"
+	lua "github.com/yuin/gopher-lua"
```
Comment (serprex) [inline flow/activities/flowable_core.go:14]: you should fix your autoformat config settings to not do this

Comment (iskakaushik) [reply flow/activities/flowable_core.go:14]: @Amogh-Bharadwaj  can you share how you did this for my edification?

Comment (iskakaushik) [reply flow/activities/flowable_core.go:14]: i spent a good 30 mins unable to figure this out. Landing as is as this is a minor thing.

**Pull request merged**